frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -143,19 +173,26 @@ export const FormComponentConfig = ({
             />
           );
         }
-        if (rest[propertyKey].type === 'string') {
+        if (
+          rest[propertyKey].type === 'string' ||
+          rest[propertyKey].enum ||
+          rest[propertyKey].$ref?.endsWith(getExpressionSchemaDefinitionReference('string'))
+        ) {
           return (
             <EditStringValue
               component={component}
               handleComponentChange={handleComponentUpdate}
               propertyKey={propertyKey}
               key={propertyKey}
               helpText={rest[propertyKey]?.description}
-              enumValues={rest[propertyKey]?.enum}
+              enumValues={rest[propertyKey]?.enum || rest[propertyKey]?.examples}
             />
           );
         }
-        if (rest[propertyKey].type === 'array' && rest[propertyKey].items?.type === 'string') {
+        if (
+          rest[propertyKey].type === 'array' &&
+          (rest[propertyKey].items?.type === 'string' || rest[propertyKey].items?.enum)
```

Comment:
Is the questionmarks needed? Wont all properties with array type also have the item prop?